app/src/main/kotlin/com/wire/android/ui/common/textfield/WireTextFieldLayout.kt

```diff
@@ -101,8 +109,8 @@ internal fun WireTextFieldLayout(
                 .fillMaxWidth()
                 .background(color = colors.backgroundColor(state).value, shape = shape)
                 .border(width = dimensions().spacing1x, color = colors.borderColor(state, interactionSource).value, shape = shape)
-                .semantics {
-                    (labelText ?: placeholderText ?: descriptionText)?.let {
+                .semantics(mergeDescendants = true) {
+                    (semanticDescription ?: labelText ?: placeholderText ?: descriptionText)?.let {
```

Comment:
same as above moving the default values to the function parameters makes it easier to understands what is going on + makes it able to set it to null when there is a need to do so

Reply:
answered to this in here https://github.com/wireapp/wire-android/pull/3541#discussion_r1812376271